packages/browser-tracker/README.md

```diff
@@ -41,6 +41,19 @@ window.elasticAnalytics.createTracker({
 });
 ```
 
+
+### Introducing sampling
+You don't always want all sessions to be sent to your Elastic cluster. You can introduce session-based sampling by adding `sampling` parameter to the `createTracker` method.
+
+If sampling is set to 1 (default), all sessions will send events. If sampling is set to 0, no sessions will send events.
+
+```js
+window.elasticAnalytics.createTracker({
+  ...
```

Comment:
nitpick: expand the ... to be:
`// ... tracker settings`

Just so its clearer they wont add ... and have a invalid syntax error

Reply:
Fixed!